commercial-paper/README.md

```diff
@@ -87,7 +87,15 @@ docker exec cliMagnetoCorp peer chaincode install -n papercontract -v 0 -p /opt/
 docker exec cliMagnetoCorp peer chaincode instantiate -n papercontract -v 0 -l java -c '{"Args":["org.papernet.commercialpaper:instantiate"]}' -C mychannel -P "AND ('Org1MSP.member')"
 ```
 
-> If you want to try both a Java and JavaScript Contract, then you will need to restart the infrastructure and deploy the other contract.
+**For a Go Contract:**
+
+```
+docker exec cliMagnetoCorp bash -c "cd /opt/gopath/src/github.com/hyperledger/fabric-samples/commercial-paper/organization/magnetocorp/contract-go GO111MODULE=on GOCACHE=on go mod vendor"
+docker exec cliMagnetoCorp peer chaincode install -n papercontract -v 0 -p github.com/hyperledger/fabric-samples/commercial-paper/organization/magnetocorp/contract-go -l golang
+docker exec cliMagnetoCorp peer chaincode instantiate -n papercontract -v 0 -l golang -c '{"Args":["org.papernet.commercialpaper:instantiate"]}' -C mychannel -P "AND ('Org1MSP.member')"
```

Comment:
This gave me:
Error: could not assemble transaction, err proposal response was not successful, error code 500, msg Channel 'mychannel' has been migrated to the new lifecycle, LSCC is now read-only
which made me realize that basic-network was upgraded to use the new cc lifecycle but commercial-paper (and maybe others) haven't. :-(

Reply:
@lehors  how did you fixed this issue? I got the same issue when init my new chaincode on test-network

Reply:
@trnhgquan These instructions have changed. When I use the latest README with Fabric 2.1 everything works (modulo some simple doc error I'm going to fix).